inference-engine/src/mkldnn_plugin/nodes/mkldnn_quantize_node.cpp

```diff
@@ -1014,21 +1014,39 @@ void MKLDNNQuantizeNode::init() {
 
         bool quantizationOnly = true;
 
-        for (int i = 0; i < cropLow.size(); i++) {
-            float il = inputLowData[isInputLowBroadcasted ? 0 : i];
+        auto prepareAlignedData = [](float *newData, const float *oldData, int dataSize, bool isBroadcasted) {
```

Comment:
Please add the commend why we need additional copy.

Reply:
added